r_basics_practice/basics_practice_exercises.Rmd

```diff
@@ -0,0 +1,92 @@
+---
+title: "R Basics Practice"
+output: html_document
+---
+
+## Step 1: Read in the data using `read_csv()`
+
+Save it as a new data frame called `yaounde_covid_seroprev`.
+
+The dataset is saved online, not locally in this folder. Its location is "https://raw.githubusercontent.com/kendavidn/yaounde_serocovpop_shared/v1.0.0/data/yaounde_covid_seroprev_dataset.csv"
+
+HINT: Remember to load the `tidyverse` package before trying to use the `read_csv()` command.
+
+```{r}
+
+```
+
+## Step 2: Inspect the data using the Data Viewer.
+
+Open the `yaounde_covid_seroprev` data in the Data Viewer.
+Answer the following questions:
+
+- How many observations (rows) are in the data? How many columns?
+- Sort the data by `val_age`. How old is the youngest person in the data? How old is the oldest?
+
+HINT: Can't find the `val_age` column? Note that RStudio only shows 50 columns at a time, so you may need to click over to the next 50 columns (columns 51-100) to find what you're looking for. Look for where it says "Cols: 1-50" at the top.
+
+## Step 3: Use dplyr to save a version of the data that will be easier to work with
+
+This dataset includes many columns that we don't actually need for our current analysis. 
+Use dplyr to save a new version of the data called `data_cleaned` that selects only the following columns: `val_age`, `had_symp_none`, `cat_igm_result`, and `cat_igg_result`. 
+
```

Comment:
Good catch! This should have been covered clearly in Intro, but it wasn't! Rather than add explanation here, I think we should add a section to Intro: https://github.com/arcus/education_modules/issues/807 
I did put in little "hints" throughout the exercises file reminding learners of where they will need to add code chunks (but not explaining how to do that).